src/dotlegendgraph.cpp

```diff
@@ -47,7 +47,7 @@ void DotLegendGraph::computeTheGraph()
   QCString fontName = Config_getString(DOT_FONTNAME);
   TextStream md5stream;
   writeGraphHeader(md5stream,theTranslator->trLegendTitle());
-  md5stream << "  node [shape=\"box\",fontsize=\"" << fontSize << "\",height=0.2,width=0.4,fontname=\"" << fontName << "\"];\n";
+  md5stream << "  node [fontsize=\"" << fontSize << "\",height=0.2,width=0.4,fontname=\"" << fontName << "\"];\n";
```

Comment:
Are "<..>" used there at all?

Reply:
It is possible, on Cygwin I did:
```
mkdir "some<weird>test"
```
and wit `ls -ls` I got:
```
   0 drwxr-xr-x+ 1 MY My         0 Jun  5 11:42 'some<weird>test'
```

so in principle it is possible ...

Reply:
It looks OK:
![Screenshot from 2022-06-05 13-04-06](https://user-images.githubusercontent.com/2335185/172045373-2c9bfcb0-a028-41d4-ab98-bcd0d0475d71.png)

Reply:
This looks OK, but so did in the past the other occurrences of `<` and `>` but as there is the `box` part this might explain that it looks OK.

In my `dot` file I see:
```
dir_f96aea8804ccd99ef1e5f41be0607cc1 [shape=box, label="some<weird>test", style="filled,bold,", fillcolor="#edf0f7", color="grey25", URL="dir_f96aea8804ccd99ef1e5f41be0607cc1.html"];
```
so this looks to me like a `node`.

My example: [example.tar.gz](https://github.com/doxygen/doxygen/files/8839770/example.tar.gz)

Reply:
Would you like to review and approve this PR and fix `drawDirectory` and `drawClusterOpening` in another PR?

Reply:
I think they should be part of this proposed pull request as they are the same type of problem (as far as I can see / understand it).

Reply:
> I think they should be part of this proposed pull request as they are the same type of problem (as far as I can see / understand it).

Would you like to review?